config/initializers/devise.rb

```diff
@@ -216,7 +216,7 @@
   # Email regex used to validate email formats. It simply asserts that
   # one (and only one) @ exists in the given string. This is mainly
```

Comment:
should this comment be updated? seems like perhaps it's not entirely accurate now

Reply:
Ah, good call. Updated!